server/player.lua

```diff
@@ -1081,29 +1068,62 @@ exports('SaveOffline', SaveOffline)
 
 ---@param identifier Source | string
 ---@param key string
+---@param subKeys? string[]
 ---@param value any
-function SetPlayerData(identifier, key, value)
+---@param cb? function A function that's called after the standard SetPlayerData events are triggered if the player is online
+---@param cancelDbUpdate? boolean When true, makes sure the database doesn't get updated as a result of this change
+function SetPlayerData(identifier, key, subKeys, value, cb, cancelDbUpdate)
```

Comment:
How about we accept a `key` parameter of both type string and a table of strings instead of introducing the `subKeys` parameter?

Reply:
Sounds better indeed, I'll apply that